portfolio/src/main/webapp/utils.js

```diff
@@ -0,0 +1,41 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/** 
+ * Set all elements of given className name to style.display = none.
+ * @param {string} name - The name of the class that I need to modify.
+ */
+const setToNone = (name) => {
```

Comment:
This should also be wrapped in a class. You can make these functions static so you can call them like Utils.setToNone().

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static